core-ui/src/shared/components/EventSubscriptionConditionStatus.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { StatusBadge } from 'react-shared';
+
+export const EventSubscriptionConditionStatus = ({ condition }) => {
+  const { t, i18n } = useTranslation();
+  const statusBadgeProperties =
+    condition?.status === 'True'
+      ? {
+          type: 'success',
+          text: t('common.statuses.ready'),
+          tooltipMessage: condition?.type,
+        }
+      : {
+          type: 'error',
+          text: t('common.statuses.error'),
+          tooltipMessage: condition?.message,
+        };
+  return (
+    <StatusBadge
+      type={statusBadgeProperties.type}
+      additionalContent={statusBadgeProperties.tooltipMessage}
+      i18n={i18n}
```

Comment:
Let's add a translation to the `ready` state, instead of getting it from the condition type. 
To do that, remove the `condition?.type` from the tooltipMessage above, add `resourceKind='event-subscription'` here, and add a transtaltion to the `en.yaml` file.